js/scripts.js

```diff
@@ -60,62 +65,227 @@ var TxtType = function(el, toRotate, period) {
+                        "<i class='fa fa-chevron-left fa-2x'></i>",
+                        "<i class='fa fa-chevron-right fa-2x'></i>"
+                     ],
+                    responsive:{
+                        0: {
+                            items: 1,
+                            // dots:false
+                        }, 
+                        485: {
+                            items: 2,
+                            // dots: false
+                        },
+                        728: {
+                            items:3,
+                            // dots:true
+                        },
+                        960: {
+                            items:3,
+                            // loop: true,
+                            
+                        },
+                        1200: {
+                            items:5,
+                            // dots:true,
+                            // loop: false,
+                            // nav: true,
+                        }
+                    }
+                    
+                });
+
+                $('owl-carousel').on('mousewheel', 'owl-stage', function(e){
+                    if(e.deltaY>0) {
+                        $('.owl-carousel').trigger('next-owl');
+                    } else  {
+                        $('owl-carousel').trigger('prev.owl');
+                    }
+                    e.preventDefault();
+                });
+
             console.log(htmlRepresetation);
-    })
-
-
-// fetch(api_url);
-// const data = await response.json();
-// console.log(data);
-
-// const contentDiv = document.getElementById('content');
-// const apiPromise = fetch("https://...")
-
-// // apiPromise
-//     .then(result => result.json())
-//     .then(content => {
-//         console.log(content);
-//         const htmlRepresetation = `
-//         <ul>
-//             <li>Name: ${content.name}</li>
-//             <li>Birthday: ${content.birthday} </li>
-//             <li> Favourite hobby: ${content.favouriteHobby}</li>
-//         </ul>
-//         `;
-//         return htmlRepresetation;
-
-//     })
-//     .then(htmlRepresetation =>{
-//         contentDiv.innerHTML = htmlRepresetation;
-//         console.log(htmlRepresetation);
-//     })
+        
+    });
+
+
+
+    /*Creating google map*/
+    function initMap() {
+        // The location of Munich
+        const munich = { lat: 48.137154, lng: 11.576124 };
+        // The map, centered at Munich
+        const map = new google.maps.Map(document.getElementById("map"), {
+          zoom: 15,
+          center: munich,
+        });
+        // The marker, positioned at Munich
+        const marker = new google.maps.Marker({
+          position: munich,
+          map: map,
+        });
+      }
+
+/**Connecting Map Location API  */
+
+// const StoresList = document.getElementById("stores-list");
+const LocationApiUrl = fetch('https://www.edeka.de/api/marketsearch/markets');
```

Comment:
as I wrote earlier - this is not a url :)
url is 'https://www.edeka.de/api/marketsearch/markets' and this is fetching data what is in your constant.
So I can suggest to store only url in a variable and then below have
```js
fetch(apiUrl)
  .then()
  .then()
```